assets/js/common/ActionsButton/ActionsButton.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef, Fragment } from 'react';
+import classNames from 'classnames';
+import { some } from 'lodash';
+import {
+  Menu,
+  MenuButton,
+  MenuItems,
+  MenuItem,
+  MenuSeparator,
+} from '@headlessui/react';
+import { EOS_MORE_VERT, EOS_LOADING_ANIMATED } from 'eos-icons-react';
+
+import Button from '@common/Button';
+
+function ActionsButton({ actions }) {
```

Comment:
Overall question, do we want common components to be more readable and destruct the required data fields or keep `actions` as it is more or less simple. But requires us to have a deeper look at the component.

Reply:
How can you destruct array element values in the signature?

Reply:
No you can't directly, or it would create more things to do. It makes here sense to have the array.